### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by VargBurz**
first version of readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
 # hastic-telegram-bot
 Tool for delivering notifications from Hastic's webhooks to Telegram
+
+This tool uses some port, that is indicated in config.
```
Comment (rozetko) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,17 @@
 # hastic-telegram-bot
 Tool for delivering notifications from Hastic's webhooks to Telegram
+
+Hastic analytic unit webhooks get in this port and handled in a certain way.
```
Comment (jonyrock) [inline README.md]: This sentence has no info. We can just delete it and nothing would change.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,17 @@
...
+Fields like message and image are extracted from webhooks and sent to the telegram chats. (Chats are indicated in config too).
```
Comment (jonyrock) [inline README.md]: > Chats are indicated in config too
In what sense do you use "indicated"? I think we can just delete this sentence.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,17 @@
...
+Telegram chat is selected according to url of webhook.
```
Comment (jonyrock) [inline README.md]: please just remove this sentence

File: README.md [hunk 5]
```diff
@@ -1,2 +1,17 @@
...
+
+Example of config :
```
Comment (jonyrock) [inline README.md]:
```suggestion
Example config:
```

**Pull request merged**